scripts/core/ArticlesListByQuery.tsx

```diff
@@ -106,11 +109,7 @@ class ArticlesListByQueryComponent extends React.PureComponent<IPropsInner, ISta
                     <ArticlesListV2
                         itemCount={itemCount}
                         pageSize={this.props.query.max_results}
-                        loadItems={(from, to) => this.loadItems(from, to).then(({_items}) => {
-                            this.markAsInitializedOnce();
-
-                            return _items;
-                        })}
+                        loadItems={(from, to) => this.loadItems(from, to).then(({_items}) => _items)}
```

Comment:
seems like all this does now is `loadItems={this.loadItems}`

Reply:
It destructures `_items` from the response.

Reply:
I see now